README.md

```diff
@@ -4,198 +4,25 @@
 [![GitHub issues](https://img.shields.io/github/issues/za419/CadenceBot?style=flat-square)](https://github.com/za419/CadenceBot/issues)
 [![GitHub forks](https://img.shields.io/github/forks/za419/CadenceBot?style=flat-square)](https://github.com/za419/CadenceBot/network)
 
-CadenceBot is a [Discord](https://discordapp.com/) bot, which serves as a client for [Cadence Radio](https://github.com/kenellorando/cadence).
+**CadenceBot** is a [Discord](https://discordapp.com/) bot client for a [Cadence Radio](https://github.com/kenellorando/cadence) stack. 
 
-CadenceBot is fully featured, and is functionally equivalent to the [web client](http://cadenceradio.com/). You can add CadenceBot to your Discord server by clicking [this link.](https://discord.com/api/oauth2/authorize?client_id=372999377569972224&permissions=274881252352&scope=bot)
```

Comment:
Should we keep the "You can add CadenceBot by clicking this link" bit? I think that's probably helpful for people a layer below wanting to dive right into being an IA - There's really nowhere else where we give the opportunity to add prod CadenceBot, so if we ditch this link we need to either find a better place to keep it or we're essentially saying "Don't use prod CadenceBot, you must deploy your own for your server" - Which I don't think is what we actually want to communicate.

Reply:
@za419 This would only add a bot configured with cadenceradio.com itself, correct? Maybe it would make sense if I added it as a "try it out immediately" one-click kind of thing.

Reply:
Correct. That's the same prod bot that we've been using. I agree with it as a one-click try-it-out thing - My logic basically sums up to "More people are more likely to want to use a bot than run it"...

Plus, it's a good demo. Probably people want to see the bot in action before committing to hosting and IA-ing a copy...